sys/shell/cmds/lwip_netif.c

```diff
@@ -105,6 +113,238 @@ static void _netif_list(struct netif *netif)
 #endif
 }
 
+
+#ifdef MODULE_LWIP_IPV4
+static void _usage_add4(char *cmd)
+{
+ printf("usage: %s add4 <interface> <IPv4>/<prefix>\n", cmd);
+ printf("usage: %s add4 <interface> <IPv4>/<prefix> gw <IPv4>\n", cmd);
+}
```

Comment:
Not sure, if this is some weird GitHub formatting bug or actually wrong. But please follow the coding conventions and use 4 spaces here and below for indentation.

Reply:
Thank. I will fix all warnings from static test in the following few days. I understand that I should clear the first version with legacy commands?